src/decompiler.cc

```diff
@@ -474,7 +475,7 @@ struct Decompiler {
         auto& se = *cast<StoreExpr>(n.e);
         LoadStore(args[0], n.children[0], se.offset, se.opcode, se.align,
                   se.opcode.GetParamType2());
-        return WrapBinary(args, " = ", true, Precedence::Assign);
+        return WrapBinary(args, "=", true, Precedence::Assign);
```

Comment:
So there were no calls to `WrapBinary` that don't want the spacing?

Reply:
Yup, all 3 calls had spaces before and after.